package.json

```diff
@@ -1,10 +1,6 @@
 {
   "name": "agicash",
   "private": true,
-  "sideEffects": ["./app/instrument.server.mjs"],
```

Comment:
seems like this is not needed anymore. not entirely sure why but doesn't even matter